CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.4.3 (2020-02-18)
+
+- Fixed the select field component to allow deselcting the selected item programmatically. [#54](https://github.com/blackbaud/skyux-select-field/pull/54)
```

Comment:
`deselecting` is misspelled.

But I wonder if this would be clearer if we rearrange the end just a bit. What do you think of `Fixed the select field component  to allow it to programmatically deselect items.` What do you think?

Reply:
I like it. Done.